numpyro/infer/util.py

```diff
@@ -416,8 +419,11 @@ def constrain_fun(*args, **kwargs):
     return potential_fn, constrain_fun
 
 
-def initialize_model(rng_key, model, *model_args, init_strategy=init_to_uniform(),
-                     dynamic_args=False, **model_kwargs):
+def initialize_model(rng_key, model,
+                     init_strategy=init_to_uniform(),
+                     dynamic_args=False,
+                     model_args=(),
+                     model_kwargs=None):
```

Comment:
this is the only non-doc related change.

Reply:
Could you also mark `find_valid_initial_params` as experimental too? We might need to cache `find_valid_initial_params` when we rerun MCMC with new data. This is usually pretty fast comparing to compile `body_fn` but in some cases, it took a few seconds to finish.

Reply:
I think that's marked as experimental but I'm changing the varargs to match the other utils.